monai/networks/blocks/transformerblock.py

```diff
@@ -40,6 +45,10 @@ def __init__(
             dropout_rate (float, optional): fraction of the input units to drop. Defaults to 0.0.
             qkv_bias (bool, optional): apply bias term for the qkv linear layer. Defaults to False.
             save_attn (bool, optional): to make accessible the attention matrix. Defaults to False.
+            window_size (int): Window size for local attention as used in Segment Anything https://arxiv.org/abs/2304.02643.
+                If 0, global attention used. Only 2D inputs are supported for local attention (window_size > 0).
```

Comment:
Could also add support for 3d?

Reply:
3d added in [70dc36c](https://github.com/Project-MONAI/MONAI/pull/7348/commits/70dc36c1c27de2609e34d6bb5645f2c55a60cd60)